cmd/jwt.go

```diff
@@ -132,10 +110,44 @@ func webRequestAuthenticate(req *http.Request) (*xjwt.MapClaims, bool, error) {
 		return nil, false, err
 	}
 	claims := xjwt.NewMapClaims()
-	if err := xjwt.ParseWithClaims(token, claims, webTokenCallback); err != nil {
+	if err := xjwt.ParseWithClaims(token, claims, func(claims *xjwt.MapClaims) ([]byte, error) {
+		if claims.AccessKey == globalActiveCred.AccessKey {
+			return []byte(globalActiveCred.SecretKey), nil
+		}
+		cred, ok := globalIAMSys.GetUser(claims.AccessKey)
```

Comment:
I think session token for STS really needs to be signed using the server root credentials. Otherwise, per my understanding, anyone who already have existing temporary account can create a new session token with escalated session policy.

Reply:
This is not related to that @vadmeste - this code path is used to validate only the JWT token for Prometheus which is never signed with secret key from root.